qa/TL1_jupyter_plugins/test_pytorch.sh

```diff
@@ -2,7 +2,7 @@
 
 # used pip packages
 # TODO(janton): remove explicit pillow version installation when torch fixes the issue with PILLOW_VERSION not being defined
```

Comment:
just wondering, should we try to remove the version pin on pillow?

Reply:
I will try